src/screens/Accounts/Accounts.js

```diff
@@ -39,24 +43,46 @@ import { ScrollWrapper } from 'components/Layout';
 import { PPN_TOKEN } from 'configs/assetsConfig';
 
 // utils
-import { getAccountName, getActiveAccount, getActiveAccountType } from 'utils/accounts';
-import { formatFiat, formatMoney } from 'utils/common';
+import {
+  getAccountName,
+  getActiveAccount,
+  getActiveAccountType,
+} from 'utils/accounts';
+import {
+  formatFiat,
+  formatMoney,
+} from 'utils/common';
 import { userHasSmartWallet } from 'utils/smartWallet';
 import { responsiveSize } from 'utils/ui';
-import { baseColors, fontStyles, spacing } from 'utils/variables';
+import {
+  baseColors,
+  fontStyles,
+  spacing,
+} from 'utils/variables';
 import { calculateBalanceInFiat } from 'utils/assets';
 
 // types
 import type { NavigationScreenProp } from 'react-navigation';
-import type { Assets, BalancesStore, Balances, Rates } from 'models/Asset';
-import type { Accounts, Account } from 'models/Account';
-import type { Dispatch, RootReducerState } from 'reducers/rootReducer';
+import type {
+  Assets,
+  Balances,
+  BalancesStore,
+  Rates,
+} from 'models/Asset';
+import type {
+  Account,
+  Accounts,
+} from 'models/Account';
+import type {
+  Dispatch,
+  RootReducerState,
+} from 'reducers/rootReducer';
```

Comment:
I think it became less readable now

Reply:
Well, TBH it was auto lint fixes, but will revert to one line imports.